packages/protocol/contracts-0.8/stability/interfaces/IFeeCurrencyAdapter.sol

```diff
@@ -2,22 +2,51 @@
 pragma solidity >=0.8.7 <0.8.20;
 
 interface IFeeCurrencyAdapter {
+  /**
+   * @return The address of the adapted token.
+   */
   function getAdaptedToken() external view returns (address);
 
+  /**
+   * @return The multiplier that should be used when upscaling and downscaling.
+   */
   function digitDifference() external view returns (uint96);
 
+  /**
+   * @return returns the amount that it's debited after calling debitGasFees().
```

Comment:
I would change to: returns the amount that *is* debited after calling debitGasFees() and before creditGasFees().